### PR TITLE
Newsletters: add 71 (2019-11-06)

### PR DESCRIPTION
**Pull request by harding**

### CODE REVIEW

File: _posts/en/newsletters/2019-11-06-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  used freely as custom records by any LN implementation.  The updated
+  specification also now provides guidance on how to select numbers for
+  custom record types.
+
+{% include linkers/issues.md issues="16943,3558,3556,660,3220,688" %}
+[bitcoin core 0.19.0]: https://bitcoincore.org/bin/bitcoin-core-0.19.0/
+[bolt4 non-strict rec]: https://github.com/lightningnetwork/lightning-rfc/blob/master/04-onion-routing.md#recommendation
+[news37 addrv2]: /en/newsletters/2019/03/12/#version-2-addr-message-proposed
+[news8 lowr]: /en/newsletters/2018/08/14/#bitcoin-core-wallet-to-begin-only-creating-low-r-signatures
+[naumenko addr relay]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-October/017428.html
```
Comment (bitschmidty) [inline _posts/en/newsletters/2019-11-06-newsletter.md]: nit on ordering

Comment (harding) [reply _posts/en/newsletters/2019-11-06-newsletter.md]: I don't usually put the links in any order---I just add new links to the list as I look them up, or duplicate a previous entry and tweak it as necessary.

**Comment by jnewbery** [comment]
ACK a826e8e24fbc7f73df5f3b656d4a388cb81df770 . Looks great!

File: _posts/en/newsletters/2019-11-06-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    signal to their peers whether or not they want to participate in
+    address relay.  This will avoid wasting bandwidth on clients that
+    don't want the addresses and can make it easier to determine the
+    consequences of certain network behavior related to address relay.
+
+    Two methods are proposed for allowing nodes to indicate whether or
+    not they want to participate in address relay---a per-node method
+    and a per-connection method.  The per-node method could easily build
+    on top of work already being done for addrv2 messages (see [Newsletter
+    #37][news37 addrv2]), but it would be less flexible than the the
```
Comment (adamjonas) [inline _posts/en/newsletters/2019-11-06-newsletter.md]: the the -> the

File: _posts/en/newsletters/2019-11-06-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+      person who initiates channel opening is responsible for paying
+      this amount (as they are responsible for paying all fees in the
+      current protocol), so they would probably like to keep it
+      low---but the amount must be greater than most node's minimum
+      output amount ("dust limit").  There was discussion about whether
+      or not this should be a configurable amount.
+
+    - Whether each LN payment should pay different public keys (using
+      pubkey tweaking).  Removing key tweaking was proposed to reduce
+      the amount state tracking necessary, but a concern was raised that
```
Comment (adamjonas) [inline _posts/en/newsletters/2019-11-06-newsletter.md]: amount -> amount of

File: _posts/en/newsletters/2019-11-06-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+- [C-Lightning #3220][] begins always creating signatures with a low *r*
+  value, reducing the maximum size of the signature by one byte and
+  saving about 0.125 vbytes on average per C-Lightning peer in onchain
+  transactions.  Bitcoin Core also previously adopted this change to its
+  wallet's signature-generation code (see [Newsletter #8][news8 lowr]).
+
+- [LND #3558][] synthesizes a unified policy for any case where two
+  particular nodes have multiple channels open between them and then
+  uses this synthetic policy when considering routing through any of
```
Comment (jonatack) [inline _posts/en/newsletters/2019-11-06-newsletter.md]: nit: maybe just me, but "synthetic policy" here could be read as fake/ersatz, perhaps "unified policy" or "policy"?

File: _posts/en/newsletters/2019-11-06-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+  those channels.  [BOLT4][bolt4 non-strict rec] recommends that multiple channels
+  between the same nodes should all use the same policy, but this
+  doesn't always happen, so this change tries to determine "the greatest
+  common denominator of all policies" between the nodes.  Using a single
+  policy reduces the number of routes the node needs to evaluate when
+  making a payment.
+
+- [LND #3556][] adds a new `queryprob` RPC that returns the expected
+  probability that a payment would succeed given a particular
+  destination node and amount.  This replaces functionality previously
```
Comment (jonatack) [inline _posts/en/newsletters/2019-11-06-newsletter.md]: IIUC the queryprob rpc takes source node, destination node, and amount as args.

File: _posts/en/newsletters/2019-11-06-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+  removed from the `querymc` RPC.
+
+- [BOLTs #660][] updates [BOLT1][] to reserve Type-Length-Value (TLV)
+  type identifiers less than 2<sup>16</sup> for types defined in in the
```
Comment (jonatack) [inline _posts/en/newsletters/2019-11-06-newsletter.md]: s/in in/in/

File: _posts/en/newsletters/2019-11-06-newsletter.md (hunk 5 above)
Comment (jonatack) [reply _posts/en/newsletters/2019-11-06-newsletter.md]: See https://github.com/joostjager/lnd/blob/fb57255b5c62a064c6f87565ed8c5ba805fd14a4/lnrpc/routerrpc/router.proto#L370

File: _posts/en/newsletters/2019-11-06-newsletter.md (hunk 1 above)
Comment (bitschmidty) [reply _posts/en/newsletters/2019-11-06-newsletter.md]: Sounds good

**Comment by bitschmidty** [comment]
ACK e2070615421e9a5184cecbddd94c79a29cdd4d80

**Comment by bitschmidty** [comment]
LN anchor outputs talks about remote delay
How should clients signal for address relay?

Please help test the latest Bitcoin Core RC
BOLTs #660 frees up space with TLV

Thanks to @harding for another hit today 
And to @jonatack and @adamjonas for keeping QA

**Pull request merged**